include/xeus-cling/xinterpreter.hpp

```diff
@@ -27,6 +24,12 @@
 #include "xbuffer.hpp"
 #include "xmanager.hpp"
 
+namespace cling
```

Comment:
Working with forward declarations avoid the we have to include the cling headers here.